plugins/scanners/scanoss/src/test/kotlin/ScanOssResultParserTest.kt

```diff
@@ -162,11 +162,51 @@ class ScanOssResultParserTest : WordSpec({
                                 "."
                             ),
                             "pkg:github/vdurmont/semver4j",
-                            SpdxExpression.parse("CC-BY-SA-2.0")
+                            SpdxExpression.parse("CC-BY-SA-2.0"),
+                            mapOf(
+                                "all_purls" to "pkg:github/vdurmont/semver4j"
+                            )
                         )
                     )
                 )
             )
         }
+
+        "should handle multiple PURLs by selecting first as primary and preserving all in metadata" {
```

Comment:
"should" should not be part of the test name, as the surrounding container already uses the `should` keyword`.